### DISCUSSION TITLE
Odata Group

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#7404] [Q&A]
Hello everyone good morning,

We have some difficulties with the use of Odata to find all the groups of the element maps we use

const url = `https://api.bentley.com/insights/imodels-odata/${iModelId}/changesets/${changeset}/mappings/${mappingId}/odata`;

now I don't know how long it hasn't been working but today it
returns "Not Found"

, {
"error": {
"code": "ResourceNotFound",
"message": "The requested resource was not found. Verify the API URL and the Accept header."
}
}

Trying to understand if something had changed on the APIs I searched google Odata sends me to this link

https://developer.bentley.com/apis/imodels-data/operations/mapping-odata/

which however seems to be Not Found

I can only confirm that inside the groups I can see the inserted groups, while in the "Select Group" combobox it does not work and says not found

<img width="360" alt="erroreOdataGroup" src="https://github.com/user-attachments/assets/f19ff6c3-8d79-45bb-b399-51a86936a4f5">

thanks for the help

**Comment by Guillar1** [comment]
Hello @Paolo-lab,

The iModels OData endpoint you are trying to access has been decommissioned, which is why you're receiving the "Not Found" error. 

To retrieve the data you need, you'll now have to perform full extractions and use the Reporting API OData endpoints. 
https://developer.bentley.com/apis/insights/operations/odata/

I recommend checking out this course, which provides step-by-step guidance on Grouping and Mapping and using the Reporting API:
https://developer.bentley.com/learning/courses/grouping-and-mapping/course-intro/course-intro/
